packages/extensions-sdk/vitest.config.ts

```diff
@@ -0,0 +1,8 @@
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+	test: {
+		// bump up timeout duration for create & build tests
+		testTimeout: 20000,
+	},
+});
```

Comment:
Can we do that in the test file instead? Per tests or on file level for `create-build.test.ts` (IIRC it should be possible to set it on a file level somehow)

Otherwise it will also apply to all future tests which might be added at some point.

Reply:
I was curious as well, but initially I wasn't able to notice per-test level timeout after only skimming the code examples in their docs. 

However turns out it was pretty obviously documented in writing and the type signature! TIL 😄 updated to per-test level.